admin/src/main/scala/net/wiringbits/components/widgets/Table.scala

```diff
@@ -0,0 +1,73 @@
+package net.wiringbits.components.widgets
+
+import com.alexitc.materialui.facade.materialUiCore.createMuiThemeMod.Theme
+import com.alexitc.materialui.facade.materialUiStyles.makeStylesMod.StylesHook
+import com.alexitc.materialui.facade.materialUiCore.{components => mui}
+import com.alexitc.materialui.facade.materialUiStyles.mod.makeStyles
+import com.alexitc.materialui.facade.materialUiStyles.withStylesMod.{
+  CSSProperties,
+  StyleRulesCallback,
+  Styles,
+  WithStylesOptions
+}
+import org.scalablytyped.runtime.StringDictionary
+import slinky.core.FunctionalComponent
+import slinky.core.annotations.react
+
+@react object Table {
```

Comment:
`Table` is likely going to get conflicts because it is a very common name, I'd call it `ExperimentalTableWidget` instead